Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -263,7 +266,8 @@ public final class ManifestLoader: ManifestLoaderProtocol {
         toolsVersion: ToolsVersion,
         packageKind: PackageReference.Kind,
         fileSystem: FileSystem? = nil,
-        diagnostics: DiagnosticsEngine? = nil
+        diagnostics: DiagnosticsEngine? = nil,
+        verbosity: Verbosity = TSCUtility.verbosity
```

Comment:
this API is deprecated so we should not add argument here, just pass in  TSCUtility.verbosity as default in L282